src/brasil/gov/portlets/upgrades/v1001/configure.zcml

```diff
@@ -0,0 +1,31 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    i18n_domain="brasil.gov.portlets">
+
+  <!-- Profile -->
+  <genericsetup:registerProfile
+      name="default"
+      title="brasil.gov.portlets: Versao 1001"
+      directory="profile"
+      description="Atualiza o pacote brasil.gov.portlets para versao 1001"
+      provides="Products.GenericSetup.interfaces.EXTENSION"
+      />
+
+  <!-- Upgrade -->
+  <genericsetup:upgradeSteps
+      source="1000"
+      destination="1001"
+      profile="brasil.gov.portlets:default">
+
+      <genericsetup:upgradeStep
+          title="Atualiza portal para versao 1001"
```

Comment:
provide a better description for the upgrade step and not just redundant information; AFAIK, this upgrade step is for installing the portlet on sites that doesn't have it, translate it.